app/src/main/res/values/strings.xml

```diff
@@ -161,4 +161,28 @@
   <string name="home_activity_back_dialog_exit">EXIT</string>
   <!-- ProfileListActivity. -->
   <string name="profile_list_activity_title">Profiles</string>
+  <!-- ProfileEditActivity. -->
+  <string name="profile_edit_created">Created on %s</string>
+  <string name="profile_edit_rename">Rename</string>
+  <string name="profile_edit_reset">Reset PIN</string>
+  <string name="profile_edit_delete">Profile Deletion</string>
+  <string name="profile_edit_delete_dialog_title">Permanently delete this profile?</string>
+  <string name="profile_edit_delete_dialog_message">All progress will be deleted and cannot be recovered.</string>
+  <string name="profile_edit_delete_dialog_positive">DELETE</string>
```

Comment:
We are using textAllCaps so you need not capitalize these strings.

Reply:
Updated to lowercase.